package.json

```diff
@@ -17,15 +17,15 @@
     "presystem-test": "npm run compile",
     "system-test": "mocha build/system-test --timeout 600000",
     "test": "c8 mocha --timeout=5000 build/test",
-    "docs": "compodoc src/",
+    "docs": "api-extractor run --local && api-documenter markdown --input-folder build/temp --output-folder docs",
```

Comment:
I don't think we'd want to use the markdown generated docs as a stopgap, even though I considered it.  I don't feel like doing the Markdown to HTML generation here just to have it replaced by our docfx pipeline, and it's too basic.